base/reflection.jl

```diff
@@ -851,7 +851,8 @@ end
 
 iskindtype(@nospecialize t) = (t === DataType || t === UnionAll || t === Union || t === typeof(Bottom))
 isconcretedispatch(@nospecialize t) = isconcretetype(t) && !iskindtype(t)
-has_free_typevars(@nospecialize(t)) = (@_total_meta; ccall(:jl_has_free_typevars, Cint, (Any,), t) != 0)
```

Comment:
Maybe we still need this, to preserve the `_total_meta`?

Reply:
Good catch! I believe when this PR was created it didn't have the `@_total_meta`. I now added `@_total_meta` to the definition in `base/boot.jl